src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -42,6 +57,22 @@ public Renderer(IServiceProvider serviceProvider)
             _componentFactory = new ComponentFactory(serviceProvider);
         }
 
+        /// <summary>
+        /// Constructs an instance of <see cref="Renderer"/>.
+        /// </summary>
+        /// <param name="serviceProvider">The <see cref="IServiceProvider"/> to be used when initialising components.</param>
```

Comment:
I feel bad about saying this, but usually we spell `initialising` in the American way (`initializing`).